### ISSUE TITLE
Index formula with null 2nd arg should still work

### ISSUE THREAD

**Issue by AJNandi** [handsontable/hyperformula#978]
### Description

In google sheets and excel, the index formula is able to default the 2nd argument to 1 if it is empty. Hyperformula will throw a #VALUE error. 

We should support leaving that 2nd argument blank, or update the documentation to make that limitation more clear. 

```
test("Match function on computed dates", () => {
  const engine = HyperFormula.buildFromArray(
    [
      ["1", "2", "3"],
      ["A", "B", "C"],
      ["1", "=INDEX(A2:C2, 1, MATCH(A3, A1:C1, 0))"],
      ["2", "=INDEX(A2:C2,, MATCH(A4, A1:C1, 0))"],
    ],
    {
      dateFormats: ["D/M/YY"],
      licenseKey: "gpl-v3",
    }
  );
  expect(engine.getCellValue({ sheet: 0, row: 2, col: 1 })).toEqual("A"); // Passes
  expect(engine.getCellValue({ sheet: 0, row: 3, col: 1 })).toEqual("B"); // Fails with #VALUE
});

```

**Comment by sequba** [comment]
@AJNandi Thank you for reporting this issue. You're right this behaviour should be documented. PR: #981

**Comment by sequba** [comment]
Released with **2.0.1**
